src/h5web/visualizations/line/MappedLineVis.tsx

```diff
@@ -52,7 +52,11 @@ function MappedLineVis(props: Props): ReactElement {
       return data;
     }
 
-    return data.flatMap((value, i) => [value - errors[i], value + errors[i]]);
+    return data.flatMap((value, i) => [
+      value - errors[i],
+      value,
```

Comment:
Adding `value` to the array used to compute the domain to handle boundary cases such the one shown as second point of #308.